docs/governance/overview.md

```diff
@@ -1,30 +1,25 @@
+Decisions concerning the direction of the project, such as how to spend Treasury funds, or amending the Decred Constitution or other policies, are made via proposals on Politeia.
 
-### Block voting
+## Politeia voting
 
-In each block, five tickets are selected pseudo-randomly and called to vote on-chain. It takes an average of 28 days for a ticket to be called to vote, though a ticket can be called anytime between 256 blocks after purchase (~20 hrs) to up to 40,960 blocks after purchase (~4 months). Tickets then vote to approve or reject the previous block of transactions created by a PoW miner. At least three out of five tickets called to vote must respond for the block to be considered valid. If a majority of voting tickets vote to approve the block, the block is accepted. If less than 50% of voting tickets approve the block (or there is a tie), all transactions from the previous block are removed and returned to the mempool; the PoW miner's reward is revoked. This power provides a check against malicious PoW miner behavior (e.g. mining empty blocks), as well as forming the basis of Decred’s [fork resistance](https://medium.com/decred/detailed-analysis-of-decred-fork-resistance-93022e0bcde7). After a ticket is called to vote, the DCR locked in that ticket is unlocked and returned to the purchaser’s wallet, along with the PoS reward if the ticket successfully voted. 
+Anyone may submit a proposal, and all data on [Politeia](politeia/overview.md) (proposals, comments, upvotes/downvotes) is periodically anchored into the Decred blockchain, using dcrtime. This enables users to cryptographically prove if censorship has occurred. In this way, Politeia is built around the idea of transparent censorship. Users cannot be silently censored. These votes can be about allocating money from the Treasury, or changing the Constitution or other policies.
 
-If a vote to change the consensus rules is occurring, tickets can also [vote](consensus-rule-voting/overview.md) to approve or reject the proposed change. A proposed change must be approved by 75% of non-abstaining tickets to take effect.
-
----
-
-## Off-chain voting
-
-Decisions concerning the direction of the project, such as how to spend Treasury funds, or amending the Decred Constitution or other policies, are made via proposals on [Politeia](https://proposals.decred.org/).
-
-Anyone may submit a proposal, and all data on Politeia (proposals, comments, upvotes/downvotes) is periodically anchored into the Decred blockchain, using [dcrtime](https://github.com/decred/dcrtime). This enables users to [cryptographically prove](politeia/politeia-censorship.md) if censorship has occurred. In this way, Politeia is built around the idea of _transparent censorship_. Users cannot be silently censored. 
+A Politeia vote is not changing the software itself. It is theoretical, and later an on-chain consensus rule change can happen, that is aligned with the Constitution or other policies, that were voted off-chain.
```

Comment:
Suggest removing this last line. Theres no indication that Politeia votes are on-chain or affect the network